### DISCUSSION TITLE
Confusion Matrix Re-arrange Column

### DISCUSSION THREAD

**Discussion by yusufitsari** [biolab/orange3#6024] [Q&A]
Hi wonderful developers, thanks for creating this awesome tools. 

I have a question, can we re-arrange column in confusion matrix? or can we make confusion matrix column always same every time we rerun the test? I am not finding anything about this in confusion matrix documentation. I already tried to rename classification class in alphabetical order, but the column is always changing every time i run new test.

Thanks.

**Comment by ajdapretnar** [comment]
Confusion matrix takes the results from Test and Score and is immutable. "Make confusion matrix column the same"? I believe you are referring to column names/order. It should always be the same, regardless of how many times you run it. Also, by default, it would order class values alphabetically. If you wish to change the order of the class values, you can do so in Edit Domain.
